web/src/lib/components/asset-viewer/detail-panel.svelte

```diff
@@ -384,7 +384,7 @@
           <p>{asset.exifInfo.make || ''} {asset.exifInfo.model || ''}</p>
           <div class="flex gap-2 text-sm">
             {#if asset.exifInfo?.fNumber}
-              <p>{$locale ? `ƒ/${asset.exifInfo.fNumber.toLocaleString($locale)}` : ''}</p>
+              <p>{$locale ? `ƒ/${asset.exifInfo.fNumber.toLocaleString($locale)}` : `ƒ/${asset.exifInfo.fNumber}`}</p>
```

Comment:
Everywhere else we just use the browser default value. I suggest to do the same here

```suggestion
              <p>ƒ/{asset.exifInfo.fNumber.toLocaleString($locale)}</p>
```

Reply:
Done